enterprise/backend/src/metabase_enterprise/serialization/v2/models.clj

```diff
@@ -25,7 +25,8 @@
   "The list of all models exported by serialization by default. Used for production code and by tests."
   (concat data-model
           content
-          ["FieldValues"
+          ["EmbeddingTheme"
```

Comment:
Can you explain why this model is added to `exported-models`? There are a bunch of other kinds of names here e.g. `data-model`, `content`, `inlined-models`.

I don't know what this new model should belong to.